it/neo4j/src/test/java/com/google/cloud/teleport/it/neo4j/Neo4jResourceManagerIT.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.neo4j;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.teleport.it.common.utils.ResourceManagerUtils;
+import com.google.cloud.teleport.it.testcontainers.TestContainersIntegrationTest;
+import java.util.List;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Integration tests for {@link Neo4jResourceManager}. */
+@Category(TestContainersIntegrationTest.class)
+@RunWith(JUnit4.class)
+public class Neo4jResourceManagerIT {
+
+  private Neo4jResourceManager neo4jResourceManager;
+
+  @Before
+  public void setUp() {
+    neo4jResourceManager = Neo4jResourceManager.builder("dummy").setDatabaseName("neo4j").build();
```

Comment:
Can you change `dummy` to something else? Some background https://developers.google.com/style/word-list#letter-d

Reply:
Fixed in 856d953740afeb1dd9050a0e39a052158cdf4dff